src/Uno.Extensions.Authentication.Oidc/HostBuilderExtensions.cs

```diff
@@ -17,14 +19,14 @@ public static class HostBuilderExtensions
 			return builder;
 		}
 
-		hostBuilder
-			.UseConfiguration(configure: configBuilder =>
-					configBuilder
-						.Section<OidcClientOptions>(name)
-				);
-
-
 		var authBuilder = builder.AsBuilder<OidcAuthenticationBuilder>();
+		if (authBuilder.Settings.Options == null)
```

Comment:
This block shouldn't be required, since OidcAuthenticationSettings is already registered here https://github.com/unoplatform/uno.extensions/blob/95000fa85bbf87a51c1d05ad329d96da3a0f3ac0/src/Uno.Extensions.Authentication.Oidc/HostBuilderExtensions.cs#L33